modules/openapi-generator/src/main/resources/dart/libraries/dio/serialization/built_value/class_concrete.mustache

```diff
@@ -0,0 +1,87 @@
+/// a concrete implmentation of [{{classname}}], since [{{classname}}] is not instantiable
+@BuiltValue(instantiable: true)
+abstract class ${{classname}} implements {{classname}}, Built<${{classname}}, ${{classname}}Builder> {
+  ${{classname}}._();
+
+  factory ${{classname}}([void Function(${{classname}}Builder)? updates]) = _$${{classname}};
+
+  @BuiltValueHook(initializeBuilder: true)
+  static void _defaults(${{classname}}Builder b) => b;
+  @BuiltValueSerializer(custom: true)
+  static Serializer<${{classname}}> get serializer => _$${{classname}}Serializer();
+}
+
+class _$${{classname}}Serializer implements PrimitiveSerializer<${{classname}}> {
+    @override
+    final Iterable<Type> types = const [${{classname}}, _$${{classname}}];
+
+    @override
+    final String wireName = r'${{classname}}';
+
+
+
+    @override
+    Object serialize(Serializers serializers, ${{{classname}}} object,
+        {FullType specifiedType = FullType.unspecified}) {        
+        return serializers.serialize(object, specifiedType: FullType({{classname}}))!;
```

Comment:
this causes an infinite loop which leads to StackOverflow

Reply:
I see the problem now, this is a problem with `built_value_generator` not adding serializers for parent classes even when marked in `@SerializersFor`